fleetspeak/src/server/dbtesting/dbtesting.go

```diff
@@ -1,1240 +1,52 @@
-// Copyright 2017 Google Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-// Package dbtesting contains utilities to help test implementations of
-// db.Store.
 package dbtesting
 
 import (
-	"bytes"
-	"context"
-	"fmt"
-	"io"
-	"io/ioutil"
-	"reflect"
-	"sort"
-	"testing"
-	"time"
-
-	log "github.com/golang/glog"
-	"github.com/golang/protobuf/proto"
-	"github.com/golang/protobuf/ptypes"
-
 	"github.com/google/fleetspeak/fleetspeak/src/common"
 	"github.com/google/fleetspeak/fleetspeak/src/server/db"
-	"github.com/google/fleetspeak/fleetspeak/src/server/ids"
-	"github.com/google/fleetspeak/fleetspeak/src/server/sertesting"
-
-	apb "github.com/golang/protobuf/ptypes/any"
-	tpb "github.com/golang/protobuf/ptypes/timestamp"
-	fspb "github.com/google/fleetspeak/fleetspeak/src/common/proto/fleetspeak"
-	mpb "github.com/google/fleetspeak/fleetspeak/src/common/proto/fleetspeak_monitoring"
-	spb "github.com/google/fleetspeak/fleetspeak/src/server/proto/fleetspeak_server"
+	"testing"
```

Comment:
nit: we normally keep the system imports in a block before non-system imports.

Reply:
Got it. Fixed.